publish-to-npm.sh

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env bash
```

Comment:
I was trying to keep all scripts in bin folders so far, any reason this has to be separate?